x-pack/plugins/security_solution/public/management/pages/event_filters/view/components/flyout/index.test.tsx

```diff
@@ -166,4 +183,24 @@ describe('Event filter flyout', () => {
 
     expect(onCancelMock).toHaveBeenCalledTimes(0);
   });
+
+  it('should renders correctly when id and edit type', () => {
+    const component = render({ id: 'fakeId', type: 'edit' });
+
+    expect(component.getAllByText('Update Endpoint Event Filter')).not.toBeNull();
+    expect(component.getByText('cancel')).not.toBeNull();
+    expect(component.getByText('Endpoint Security')).not.toBeNull();
+  });
+
+  it('should dispatch action to init form store on mount with id', async () => {
+    await act(async () => {
+      render({ id: 'fakeId', type: 'edit' });
+      await waitForAction('eventFiltersInitFromId');
+    });
+
+    expect(mockedContext.store.getState().management.eventFilters.form.entry).not.toBeUndefined();
```

Comment:
There are a few tests here that also access the store directly - if selectors exist, I would suggest using them instead